From 0cfde3ec1b3c285fd6a5990845b27d8e3d545258 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 18 Dec 2013 17:49:27 +0800
Subject: [PATCH 22/30] ENGR00290615 WifiDisplay:disable VPU reorder function

Indroduce new index and struct to support disable VPU reorder
feature from Nuplayer.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 include/media/hardware/HardwareAPI.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/include/media/hardware/HardwareAPI.h b/include/media/hardware/HardwareAPI.h
index de3aeb1..214bd19 100644
--- a/include/media/hardware/HardwareAPI.h
+++ b/include/media/hardware/HardwareAPI.h
@@ -157,6 +157,16 @@ struct PrependSPSPPSToIDRFramesParams {
     OMX_BOOL bEnable;
 };
 
+// A pointer to this struct is passed to OMX_GetParameter when the extension
+// index for the 'OMX.google.android.index.disableAVCReorder'
+// extension is given.  The usage bits bDisable indicates that AVC reorder
+// should be enable or disable.
+struct DisableAVCReorderParams {
+    OMX_U32 nSize;
+    OMX_VERSIONTYPE nVersion;
+    OMX_BOOL bDisable;
+};
+
 }  // namespace android
 
 extern android::OMXPluginBase *createOMXPlugin();
-- 
1.8.0

